### ISSUE TITLE
Improve Kubernetes session.start events

### ISSUE THREAD

**Issue by klizhentas** [gravitational/teleport#3528]
#### Things to be implemented
**Easy**

* [x] k8s users
* [x] k8s groups
* [x] Containername
* [x] Podname
* [x] Namespace
* [x] Timestamp
* [x] Amount of bytes of Input sent
* [x] Amount of bytes of Output received
* [x] Initial command, if provided

**Possible, but requires extra privileges for proxy and will be done**

* [x] Containerimage
* [x] Node name

**Custom metadata**

* [x] Kubernetes cluster
* [ ] Custom entries that we can specify through configuration (but are static to Teleport)

This can be done by adding labels from proxy to the audit log. Those labels can be included as a custom metadata to the audit events.

#### Out of scope 
**Not going to be implemented unless BPF feature is added to GKE**

* [ ] A list of opened files in that event, with per file: Read or Write and Path to the file

Dupe:

https://github.com/gravitational/teleport/issues/2332

**Comment by klizhentas** [comment]
Worst case: 3 mythical person/weeks
Best case: 2 mythical person/weeks
Addresses Partially: A16

**Comment by awly** [comment]
Note: this is waiting on session streaming to land.

**Comment by russjones** [comment]
Best: 1
Worst: 2

**Comment by awly** [comment]
Note to self: also add k8s cluster name to all k8s events
